misc/docerina/src/main/java/org/ballerinalang/docgen/docs/BallerinaDocGenerator.java

```diff
@@ -276,28 +250,30 @@ private static void writeAPIDocs(PackageLibrary packageLib, String output, boole
             }
         }
         // Copy docerina ui
-        File source = new File(System.getProperty("ballerina.home") + File.separator + "lib" + File.separator +
-                "tools" + File.separator + "doc-ui");
-        File dest;
-        if (source.exists()) {
-            dest = new File(output);
-            try {
-                FileUtils.copyDirectory(source, dest);
-            } catch (IOException e) {
-                out.println(String.format("docerina: failed to copy doc ui. Cause: %s", e.getMessage
-                        ()));
-                log.error("Failed to copy the doc ui.", e);
-            }
-        } else {
-            dest = new File(output, "index.html");
-            try {
-                FileUtils.copyInputStreamToFile(BallerinaDocGenerator.class
-                        .getResourceAsStream("/doc-ui/index.html"), dest);
-                //BallerinaDocUtils.unzipResources(, dest);
-            } catch (IOException e) {
-                out.println(String.format("docerina: failed to copy doc ui. Cause: %s", e.getMessage
-                        ()));
-                log.error("Failed to copy the doc ui.", e);
+        if (!excludeUi) {
+            File source = new File(System.getProperty("ballerina.home") + File.separator + "lib" +
+                    File.separator + "tools" + File.separator + "doc-ui");
```

Comment:
Use `Path` to create the paths and use `toFile` function of `Path` class to get the file.